content/content-strategy/cms/top-takeaway.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: text/callout
+---
+### [3]What you’ll get
```

Comment:
Incorrect display level. Needs to be `### [2]` to show a heading 3.

Reply:
Steve. Fixed. Removed file. Was the old topic takeaway